private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -244,13 +248,13 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.BaseOutDirPath,
 		baseOutDirPathFlagName,
 		baseOutDirPathFlagShortName,
-		".",
+		baseOutDirFlagDefault,
```

Comment:
The other flag defaults aren't put into variables because they are zero-values of their respective types